_posts/blog/2017-09-22-guide-to-cleveland-data.md

```diff
@@ -66,6 +92,7 @@ Cleveland's Open Data Census
 * state-wide geographic data (roads, railroads, addresses, parcels)
```

Comment:
Just to mention here, I think http://ogrip.oit.ohio.go is no longer available. Should I remove it as part of this pr?

Reply:
Think it's actually https://ogrip.oit.ohio.gov/ and it's just missing the v at the end. I'll add it to this.